### PR TITLE
12.13.0 release

### PR DESCRIPTION
**Pull request by chrisbreiding**

### CODE REVIEW

File: docs/guides/references/changelog.mdx [hunk 1]
```diff
@@ -2,6 +2,28 @@
 title: Changelog
 ---
 
+## 12.13.0
+
+_Released 05/23/2023_
+
+**Features:**
+
+- Adds Git-related messages for the [Runs page](https://docs.cypress.io/guides/core-concepts/cypress-app#Runs) and [Debug page](https://docs.cypress.io/guides/cloud/runs#Debug) when users aren't using Git or there are no recorded runs for the current branch. Fixes [#26680](https://github.com/cypress-io/cypress/issues/26680).
```
Comment (mschile) [inline docs/guides/references/changelog.mdx:11]:
```suggestion
- Adds Git-related messages for the [Runs page](/guides/core-concepts/cypress-app#Runs) and [Debug page](/guides/cloud/runs#Debug) when users aren't using Git or there are no recorded runs for the current branch. Fixes [#26680](https://github.com/cypress-io/cypress/issues/26680).
```

File: docs/guides/references/changelog.mdx
Comment (mschile) [inline docs/guides/references/changelog.mdx:1]: You'll probably need to run linting on this file.

File: docs/guides/references/changelog.mdx [hunk 2]
```diff
@@ -2,6 +2,49 @@
...
 
+## 12.13.0
+
+_Released 05/23/2023_
+
+**Features:**
+
+- Adds Git-related messages for the
+  [Runs page](https://docs.cypress.io/guides/core-concepts/cypress-app#Runs) and
+  [Debug page](https://docs.cypress.io/guides/cloud/runs#Debug) when users
```
Comment (mschile) [inline docs/guides/references/changelog.mdx:13]:
```suggestion
  [Runs page](/guides/core-concepts/cypress-app#Runs) and
  [Debug page](/guides/cloud/runs#Debug) when users
```

Comment (chrisbreiding) [reply docs/guides/references/changelog.mdx]: Fixed in https://github.com/cypress-io/cypress-documentation/pull/5272/commits/e482e95287ed1e3ea1cb9a58d91b661d8123da81

**Review by mschile — APPROVED** [review]

**Review by ryanthemanuel — APPROVED** [review]

**Pull request merged**